docs/proposals/kubelet-eviction.md

```diff
@@ -69,6 +69,7 @@ The `kubelet` will support the ability to trigger eviction decisions on the foll
 | Eviction Signal  | Description                                                                     |
 |------------------|---------------------------------------------------------------------------------|
 | memory.available | memory.available := node.status.capacity[memory] - node.stats.memory.workingSet |
+| disk.available   | disk.available := node.stats.fs.available |
```

Comment:
Discussed this with @vishh: my concern is that one number is hard to choose if the node is configured to have different partitions for images. If the number is too small, we end up to evict an image to get out of the out-of-disk situation, and pull another image to back to the bad state again. But I agreed with Vish today, the root fs is used for emptydir too, which could be big too. We can iterate this later.